racket.rkt

```diff
@@ -0,0 +1,2 @@
+#lang racket
```

Comment:
When trying to run this code, it seems like this line somehow prevents the code from printing. Removing this line allows the code to print an output to the terminal.
```suggestion
```

Reply:
oh, interesting. Will fix that now!